frontend/i18n-scripts/build-i18n.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+i18next "public/!(dist)**/**/*.{js,jsx,ts,tsx}" [-oc] -c "./public/i18next-parser.config.js" -o "public/locales/\$LOCALE/\$NAMESPACE.json"
```

Comment:
Where do `$LOCALE` and `$NAMESPACE` get defined? I'd expect this script to fail when you run it since you have `set -u`.

Reply:
They don't get defined in the file. I was following the parser docs here: https://github.com/i18next/i18next-parser. The locale and namespace allegedly get injected.